test/spec/modules/rubiconBidAdapter_spec.js

```diff
@@ -422,6 +449,93 @@ describe('the rubicon adapter', () => {
             expect(window.DigiTrust.getUser.calledOnce).to.equal(true);
           });
         });
+
+        describe('singleRequest config', () => {
```

Comment:
tests confirming requests are combined as expected are good, but should have some tests verifying the params are combined into the grouped request as expected, and that the `bidRequest` param is an array of all the grouped requests.